packit_service/worker/jobs.py

```diff
@@ -264,6 +270,10 @@ def process_jobs(self, event: Event) -> List[TaskResults]:
         """
         Create a Celery task for a job handler (if trigger matches) for every job defined in config.
         """
+        if get_handlers_triggered_by_comment(event):
+            # if event is triggered by `/packit` command, we require packit config file
+            # (mypy can't know that we get here only if `event` is `*Comment*` type)
+            event.fail_when_config_file_missing = True  # type: ignore
```

Comment:
I would try to avoid this. Such things can be hard to debug. And when reading the code, it's not super clear why we are getting handlers (and don't use them).